### ISSUE TITLE
App crashing on clicking to view product details 

### ISSUE THREAD

**Issue by PrajwalM2212** [openfoodfacts/openfoodfacts-androidapp#1511]
![screenshot_1524746615](https://user-images.githubusercontent.com/30733262/39306367-a4eb3c78-497d-11e8-924b-5ec112dbb3b7.png)

**Comment by teolemon** [comment]
```
04-26 14:54:52.552 25730-25730/openfoodfacts.github.scrachx.openfood E/AndroidRuntime: FATAL EXCEPTION: main
    Process: openfoodfacts.github.scrachx.openfood, PID: 25730
    java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.ProgressBar.setVisibility(int)' on a null object reference
        at openfoodfacts.github.scrachx.openfood.network.OpenFoodAPIClient.getProduct(OpenFoodAPIClient.java:130)
        at openfoodfacts.github.scrachx.openfood.views.holders.HistoryScanHolder.lambda$new$0$HistoryScanHolder(HistoryScanHolder.java:44)
        at openfoodfacts.github.scrachx.openfood.views.holders.HistoryScanHolder$$Lambda$0.onClick(Unknown Source:21)
        at android.view.View.performClick(View.java:6294)
        at android.view.View$PerformClick.run(View.java:24770)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by PrajwalM2212** [comment]
fixed in #1510
